src/dune_rules/gen_rules.ml

```diff
@@ -392,6 +410,10 @@ let gen_rules ~sctx ~dir components : Build_config.gen_rules_result Memo.t =
       >>= function
       | Group_part _ -> redirect_to_parent
       | Standalone_or_root { directory_targets; contents } ->
+        let directory_targets =
+          (* [dir] isn't quite right *)
+          collect_directory_targets sctx ~dir ~init:directory_targets
```

Comment:
@rgrinberg I don't remember, did we decide this was fine in the end?

Reply:
yes, it's fine.

Reply:
Shall I clean up that comment then?

Reply:
Yes